src/controllers/check.your.answers.controller.ts

```diff
@@ -37,7 +37,8 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
       backLinkUrl,
       templateName: config.CHECK_YOUR_ANSWERS_PAGE,
       hasTrusts,
-      appData
+      appData,
+      changeLinks: config.CHANGE_LINKS
```

Comment:
You do not need to pass it into the render, you can use the reference of `OE_CONFIGS` in the template, just use `OE_CONFIGS.CHANGE_LINKS`